### ISSUE TITLE
Raspberry Pi/ Pi2 build

### ISSUE THREAD

**Issue by MrZoidberg** [xsmart/opencvr#35]
Is it possible to run opencvr on raspberry pi or raspberry pi 2?

**Comment by xsmart** [comment]
i think it is possible, current we are split the server and client, the server can be build for any platform without GUI.

**Comment by Pdobry** [comment]
I don't think it can be built on Linux. Make tries to access scripts in non-existing ./linux directory.

**Comment by exceeded** [comment]
?No currently this build cannot be built successfully...



________________________________

**Comment by milonet** [comment]
how can compile only server in Raspberry pi 3 with raspbian?

**Comment by xsmart** [comment]
hi milonet, 
you are Marco Milone?the guy send mail to me?

**Comment by xsmart** [comment]
we have start the porting for Raspberry pi 3, and it will only have the server part.
https://github.com/xsmart/opencvr/blob/master/rules-pi3.mk

**Comment by xsmart** [comment]
Raspberry pi 3 version is ready.
https://sourceforge.net/projects/vscloud/?source=navbar
